pages/flashbar/interactive.page.tsx

```diff
@@ -34,7 +34,7 @@ export default function FlashbarPermutations() {
   };
 
   const add = (type: FlashbarProps.Type, hasHeader = false) => {
-    setItems(items => [...items, generateItem(type, dismiss, hasHeader)]);
+    setItems(items => [generateItem(type, dismiss, hasHeader), ...items]);
```

Comment:
Updated the test page to follow the new guidelines, so clicking on the buttons that add notifications adds them to the top, not the bottom.

(True that the new guidelines are not official yet, but it seems reasonable to avoid double work and change them again when releasing. And after all these are not reference demos, just tests)